packages/hooks/README.md

```diff
@@ -52,6 +52,12 @@ One notable difference between the JS and PHP hooks API is that in the JS versio
 -   `filters`
 -   `defaultHooks`
 
+
+#### Parameters
+
+- `hookName` unique name of the hook. It may contains only alphanumeric characters and dashes `[a-zA-z\-\_]`. Slashes are not permitted.
```

Comment:
hook name from doc block: Should be a non empty string containing  only numbers, letters, dashes, periods and underscores. Also,   the hook name cannot begin with `__`.